go/libraries/doltcore/env/actions/remotes.go

```diff
@@ -352,6 +354,11 @@ func FetchRemoteBranch(ctx context.Context, tempTablesDir string, rem env.Remote
 	wg, progChan, pullerEventCh := progStarter(newCtx)
 	err = FetchCommit(ctx, tempTablesDir, srcDB, destDB, srcDBCommit, progChan, pullerEventCh)
 	progStopper(cancelFunc, wg, progChan, pullerEventCh)
+	if err == nil {
+		cli.Println()
```

Comment:
I don't think we want any output here at all, right?

Reply:
it is for printing newline after printing output from changed branch